appholder/src/main/java/com/android/identity/wallet/transfer/TransferManager.kt

```diff
@@ -83,12 +89,21 @@ class TransferManager private constructor(private val context: Context) {
                 communication.setupPresentation(presentation)
             },
             onNewRequest = { request ->
+                presentationLogStore.newLogEntryWithRequest(
```

Comment:
record a new log entry with specified request, session transcript and QR engagement